docs/modules/callbacks.rst

```diff
@@ -0,0 +1,27 @@
+Callbacks
+=========
+
+.. note::
+   `Conceptual Guide <https://docs.langchain.com/docs/components/callbacks>`_
```

Comment:
This (and others, below) link should be relative, like `./callbacks/how_to_guides.html`, no?

Reply:
Same as in the other comment you left, did it this way because that's what I saw, but makes sense to replace that with the relative path to the file itself rather than the URL 👍🏻

Reply:
I checked and the conceptual guide is automatically generated so there's no actual file to point to, we can only point to the URL itself, see https://raw.githubusercontent.com/hwchase17/langchain/master/docs/modules/agents/agents.rst or any other file under `docs/modules/` for reference 👍🏻